compiler/haskell-ide-core/src/Development/IDE/Types/Diagnostics.hs

```diff
@@ -167,20 +161,61 @@ prettyDiagnostic (fp, LSP.Diagnostic{..}) =
     where
         sev = fromMaybe LSP.DsError _severity
 
-prettyDiagnosticStore :: DiagnosticStore -> Doc SyntaxClass
```

Comment:
These functions were simply unused.

Reply:
Yes, I had noticed this, but wondered if DMD was going to use them in a future version...